RELEASE_NOTES.md

```diff
@@ -169,17 +169,16 @@ After all the backups and changes mentioned above have been completed, follow th
 Then run the below command, and replace all the configuration with the one that Decidim is providing by default:
 
 ```console
+rm -r config/webpack/* config/webpacker.yml
 bundle exec rake decidim:webpacker:install
 ```
```

Comment:
It seems this was caused by the way Shakapacker works. Since we did not had a config file at that time, Shaka did not registered the rake task.

I changed the way we perform the upgrade, and it should be much smoother now.